services/gitea/9.2.0/defaults/cm.yaml

```diff
@@ -62,12 +73,11 @@ data:
       readOnly: true
       mountPath: "/git-tls"
     clusterDomain: cluster.local.
-    memcached:
+    redis-cluster:
       priorityClassName: "dkp-critical-priority"
-      image:
-        tag: 1.6.15-debian-11-r8
+    postgresql-ha:
+      enabled: false
     postgresql:
+      enabled: true
       primary:
         priorityClassName: "dkp-critical-priority"
-      image:
-        tag: 11.16.0-debian-11-r9
```

Comment:
in the last major gitea bump, we explicitly wanted to pin this postgres version to avoid having to deal with upgrading to postgres 15, which is a non-trivial task to automate.

Reply:
ahh good call, I couldn't figure out the reasoning behind why it was pinned :)